backend/src/schemas/profile.schema.ts

```diff
@@ -2,21 +2,86 @@ import { Prop, SchemaFactory, Schema } from '@nestjs/mongoose';
 import { ApiProperty } from '@nestjs/swagger';
 import { HydratedDocument } from 'mongoose';
 
+@Schema()
+class MetaData {
+  @Prop({ required: true })
+  @ApiProperty({ example: 12 })
+  totalCoin: number;
+
+  @Prop({ required: true })
+  @ApiProperty({ example: 100 })
+  missionSuccess: number;
+}
+
+@Schema()
+class Character {
+  @Prop({ required: true })
+  @ApiProperty({ example: 'id' })
+  id: string;
+
+  @Prop({ required: true })
+  @ApiProperty({ example: 'name' })
+  name: string;
+
+  @Prop({ required: true })
+  @ApiProperty({ example: 'photo_url' })
+  photoUrl: string;
+
+  @Prop({ required: true })
+  @ApiProperty({ example: 'detail' })
+  detail: string;
+}
```

Comment:
please move `Character` schema to new file

Reply:
fixed